src/styles/sections/hero/hero.scss

```
@@ -85,3 +85,76 @@ $fade-height: 200px;
     opacity: 1;
   }
 }
+
+.subscribe {
+  width: 100%;
+  margin-top: 2rem;
+  font-size: 1.125rem;
+  font-weight: 600;
+  text-align: center;
+
+  &__title {
+    margin: 0;
+    color: $c-primary;
+    font-size: 2rem;
+    font-weight: 800;
+    text-transform: uppercase;
+  }
+  &__subtitle {
+    margin: 0 0 2rem;
+  }
+
+  &__email {
+    width: 100%;
+    max-width: 24rem;
+    box-sizing: border-box;
+    padding: 0 1rem;
+    margin: 0 auto;
+    text-align: left;
+
+    &-label {
+      display: block;
+      padding-bottom: 4px;
+      padding-left: 8px;
+    }
+    &-input {
+      display: block;
+      width: 100%;
+      box-sizing: border-box;
+      padding: 10px 20px;
+      border: 3px solid #3f5081;
+      backdrop-filter: blur(2px);
+      background-color: rgb(255 255 255 / 67%);
```

Email Input in solid white would be a great improvement. (As we accorded)
```suggestion
      background-color: rgb(255 255 255);
```

---

Done